test/types/records/split-init/split-init-type-multiple.chpl

```diff
@@ -0,0 +1,12 @@
+config const setting = false;
```

Comment:
Would this work if `setting` were a `param`?  Should there be a test that locks that behavior in, either way?